internal/cli/atlas_dbusers_create.go

```diff
@@ -16,7 +16,8 @@ package cli
 
 import (
 	"errors"
-	"strings"
+
+	"github.com/mongodb/mcli/internal/convert"
```

Comment:
[nit] sometimes importfmt misses the cases where there's an extra empty line, please delete the line and re run `make fmt`